b3lb/rest/b3lb/lb.py

```diff
@@ -57,6 +60,20 @@ def check_meeting_existence(meeting_id, secret):
             return get_node_params_by_lowest_workload(secret.tenant.cluster_group), True
 
 
+def check_parameter(params, tenant):
+    parameters = Parameter.objects.filter(tenant=tenant)
+    for parameter in parameters:
+        if parameter.parameter in params:
+            if parameter.mode == Parameter.BLOCK:
+                del params[parameter.parameter]
+            elif parameter.mode == Parameter.OVERRIDE:
+                params[parameter.parameter] = parameter.value
+        elif parameter.mode == Parameter.SET:
```

Comment:
This should be `parameter.mode in [Parameter.SET, Parameter.OVERRIDE]`, shouldn't it? Overriding should also set the parameter if the original request did not contain it.

Reply:
fixed